### DISCUSSION TITLE
How can I merge translations coming via a middleware from backend

### DISCUSSION THREAD

**Discussion by christianholland** [nuxt-modules/i18n#2733] [Q&A]
### Environment

------------------------------
- Operating System: Windows_NT
- Node Version:     v18.18.1
- Nuxt Version:     3.5.3
- Nitro Version:    2.5.2
- Package Manager:  npm@9.8.1
- Builder:          vite
- User Config:      ssr, app, runtimeConfig, alias, imports, css, build, modules, vue, vite  
- Runtime Modules:  @nuxtjs/i18n@8.0.0, @pinia/nuxt@0.4.7, ()
- Build Modules:    -
------------------------------

### Reproduction

NA

### Describe the bug

Hi there,

when a user enters my app some translations are loaded via a middleware from a server. I would like to use the function `mergeLocaleMessage()` to expand my translations with those coming from the server. However, within my middleware (`*.ts`) I cannot get access to i18n

```
const { mergeLocalMessage } = useI18n()

```

I get the error 
```
Must be called at the top of a `setup` function
```

Any help how I could get access to i18n within a middleware would be highly appreciated

**Comment by kazupon** [comment]
Thank you for your reporting!
Your issue is not bug.
so, I'll move to github discussions.
